src/otx/core/data/entity/base.py

```diff
@@ -27,6 +27,7 @@ class ImageInfo(tv_tensors.TVTensor):
         img_id: Image id
         img_shape: Image shape (heigth, width) after preprocessing
         ori_shape: Image shape (heigth, width) right after loading it
+        attributes: Image attributes
```

Comment:
This seems a variable only for tiling to me. Can we move this to `Tile*DataEntity`? We don't want to increase the size of variables in the base object indefinitely.